va/va.h

```diff
@@ -1137,6 +1142,17 @@ typedef union _VAConfigAttribValMultipleFrame {
     uint32_t value;
 }VAConfigAttribValMultipleFrame;
 
+/** brief Attribute value VAConfigAttribValContestPriority */
+typedef union _VAConfigAttribValContextPriority{
+    struct{
+        /** \brief the priority , for the Query operation it will return the range of priority */
```

Comment:
I assume this "for the Query operation it will return the range of priority" sounds weird. We don't have **ranges** here, we have only **one value**. If it's 'max' I propose to say it explicitly in the code comment that range is between [0, priority].